BridgeAppSDK/SBAActivityTableViewController.swift

```diff
@@ -156,6 +167,28 @@ public class SBAActivityTableViewController: UITableViewController, SBASharedInf
     
     // MARK: Protected subclass methods
     
+    public func shouldShowTaskForSchedule(schedule: SBBScheduledActivity) -> Bool {
+        return true
```

Comment:
Override, I think. (WIP) Need to be able to conditionally override to allow Lilly to disallow doing the activity *again* for some cases but *not* others. I'll add a comment.